### ISSUE TITLE
[bug] Skipping goals -Dskip.npm and -Dskip.webpack doesn't work

### ISSUE THREAD

**Issue by AndreyRubankov** [eirslett/frontend-maven-plugin#595]
As a developer I want to be able to skip _npm install_ step through -Dskip.npm=true.

Steps to reproduce:
1. Run the command: mvn clean install -Dskip.npm=true -Dskip.webpack=true

Expected: The npm step and the webpack step will be skipped.
Actual: Both steps run.

Plugin Versions: 1.3, 1.4, 1.5-SNAPSHOT
OS: MacOs
Java1.8
Maven: 3.2.5, 3.5.0

**Comment by AndreyRubankov** [comment]
I've investigated this issue and found when this issue can be fixed.

It's looks pretty simple: change default value from hardcoded 'false' to EL tag with property name.
```
@Parameter(property = "skip.npm", defaultValue = "${skip.npm}")
private boolean skip;
```
In my local tests it was fixed this issue.

@eirslett , If you agree with this solution I'll create patch little bit later.

**Comment by eirslett** [comment]
Yes, that sounds like a good idea.

**Comment by AndreyRubankov** [comment]
PR for fix has been created: https://github.com/eirslett/frontend-maven-plugin/pull/596 .

Could you please review it and merge?

**Comment by AndreyRubankov** [comment]
Thank you Eirik,
So, for history: this issue will be fixed in version 1.5.
